### ISSUE TITLE
Bump contrast ratio for surname on about page

### ISSUE THREAD

**Issue by patricknelson** [syntaxfm/website#1249]
Just a quick suggestion for A11y. See below. "Bos" (on initial state) and "Vinegar" on hover states are difficult to read. Unfortunately DevTools cannot calculate contrast ratio when on top of images I guess, so I just fudged it a bit and used a color picker to get  a value of #E9DDDD (from the wall behind the head) to help demonstrate. This had a value of `1.32`.  https://www.siegemedia.com/contrast-ratio#%23fff-on-%23E9DDDD

Maybe some sort of offset, shadow or even another background might improve the contrast a bit. 😎

![image](https://github.com/syntaxfm/website/assets/4269377/a8565123-4dd9-4ec4-94f2-113fdb3ce9e6)

![image](https://github.com/syntaxfm/website/assets/4269377/328e29b6-4d6f-43a2-8cfb-d0012ea2a741)

**Comment by wesbos** [comment]
thanks! Fixed it for Bens photo
